uxarray/grid.py

```diff
@@ -304,7 +304,7 @@ def compute_face_areas(self, quadrature_rule="triangular", order=4):
             if not "degree" in self.Mesh2_node_x.units:
                 coords_type = "cartesian"
 
-            face_nodes = self.Mesh2_face_nodes.data
+            face_nodes = self.Mesh2_face_nodes.data.astype(np.int64)
```

Comment:
Explicitly setting face nodes as integers is great, however there are some small inconsistencies throughout the code (some sections call for `np.int64` which others only `int`)